examples/python/template/main.py

```diff
@@ -0,0 +1,44 @@
+#!/usr/bin/env python3
+"""
+Example template.
+
+Run:
+```sh
+pip install -r examples/python/template/requirements.txt
+./examples/python/template/main.py
+```
+"""
+from __future__ import annotations
+
+import argparse
+import logging
+
+import rerun as rr  # pip install rerun-sdk
+
+
+def setup_logging() -> None:
+    # Forward all text logs to Rerun by attaching a handler directly to the root logger,
+    # which will catch events from all loggers going forward (propagation is on by
+    # default).
+    #
+    # For more info: https://docs.python.org/3/howto/logging.html#handlers
+    logging.getLogger().addHandler(rr.log.text.LoggingHandler())
+    logging.getLogger().setLevel(-1)
```

Comment:
I think we should either remove this or clarify that this is just an example of what you can do in the case where you want to get python logs into Rerun

Reply:
Removed it as its probably too specific for an example template